pkg/v1dbschema/schema.go

```diff
@@ -152,6 +152,16 @@ func updateSyncerOnlineDDLMeta(tctx *tcontext.Context, dbConn *conn.BaseConn, ta
 
 // getGlobalPos tries to get the global checkpoint position.
 func getGlobalPos(tctx *tcontext.Context, dbConn *conn.BaseConn, tableName, sourceID string) (gmysql.Position, error) {
+	failpoint.Inject("MockGetGlobalPos", func(val failpoint.Value) {
```

Comment:
For this kind of function, is it easier and more flexible to depend on sqlmok

Reply:
fixed in  https://github.com/pingcap/dm/pull/2096/commits/60f56dbc3048c5407570aeb8e548b38c400053fd 😬